webpack/assets/javascripts/react_app/components/Layout/components/CustomContextSelector.js

```diff
@@ -0,0 +1,241 @@
+/**
+ * Modified PF4 ContextSelector
+ *
+ * Includes static group before the filter input and can show the filter input conditionally
+ * Removed FocusTrap wrapper
+ */
+import React from 'react';
+/* eslint-disable */
+import styles from '@patternfly/react-styles/css/components/ContextSelector/context-selector';
+import selectStyles from '@patternfly/react-styles/css/components/Select/select';
+import { css } from '@patternfly/react-styles';
+import SearchIcon from '@patternfly/react-icons/dist/js/icons/search-icon';
+/* eslint-enable */
```

Comment:
This will likely fail to build once the component is used anywhere, see https://github.com/theforeman/foreman/pull/7726#issuecomment-662938531 and https://github.com/theforeman/foreman/pull/7850

Reply:
@sharvit can we add `@patternfly/react-styles` to foreman-js or foreman? will it solve this?

Reply:
Yes, we should be able to add it to the vendor

Reply:
I'll add it to vendor and mark the pr that uses CustomContext as blocked

Reply:
Would it make sense to also remove the `eslint-disable` in that pr as well so we make sure it's resolvable?

Reply:
:+1: